pkg/upload/ftp.go

```diff
@@ -186,6 +187,23 @@ func (agent *FTPTransferAgent) ReturnPath() string {
 	return agent.cfg.ReturnPath
 }
 
+func (agent *FTPTransferAgent) Hostname() string {
+	if agent.cfg.FTP == nil {
+		return ""
+	}
+
+	host := agent.cfg.FTP.Hostname
+	if !strings.Contains(host, ":") {
```

Comment:
Is there a reason we'd drop the port? It might be helpful to include that - I didn't think about stripping the port in the original issue. 

I've found that on-call notifications should include exactly the config/error occurring. Modify a config value like this has caused me to dig into the wrong thing - what if the port really is missing or misconfigured?

Reply:
I can include the port.